src/js/store.js

```diff
@@ -52,7 +52,7 @@ export const defaultShortcuts = {
 };
 
 // Settings
-export const language = SS('language', 'English');
+export const languages = SS('languages', ['English']);
```

Comment:
This will cause migration errors as some people have already selected a language other than English and the choice will not be reflected in the updated app. Can you make it so that the `language` store contains the primary language (the one to be used in the translator mode auto-prefix) and make the `languages` store update its value based on the initial `language` store value? I was thinking something along the lines of

```js
await language.loaded;
await hasDoneLanguageMigration.loaded;
if (!hasDoneLanguageMigration) {
  languages.set([language.get()]);
}
```

Reply:
@r2dev2 Ok can you check out the changes I made I just used the empty languages store instead of a separate store to detect migration. Not sure if there's an issue with that or not so lmk what you think.

Reply:
I think that a separate store which stores whether the migration happened would work better as the user may want to get no translations from YouTube chat.